mmdet/models/necks/fpn.py

```diff
@@ -85,13 +85,14 @@ def __init__(self,
         self.fp16_enabled = False
         self.upsample_cfg = upsample_cfg.copy()
 
-        if end_level == -1:
+        if end_level == -1 or end_level == self.num_ins:
```

Comment:
if end_level == -1 or end_level == (self.num_ins - 1):
for end_level is the Index of the end input backbone level.